draft-ietf-mls-protocol.md

```diff
@@ -2200,6 +2200,16 @@ from the latest ratcheting tree. If the adversary cannot derive the
 updated root key after an Update operation, it cannot compute any
 derived secrets.
 
+## State loss and recovery
+
+In case part of the group messaging state is lost, and if enough
+information about the group is still available, a client MUST perform
+an update of their leaf secret before sending application messages.
+
+In the case where the client cannot recover from the state loss,
+the client SHOULD signal the delivery service to expire all the
+previous ClientInitKeys and publish fresh ones to avoid replay attacks.
+
```

Comment:
Note that this will be definitely needed to recover PCS if we do the signature key rotation.